src/server.ts

```diff
@@ -98,6 +106,322 @@ class REMSServer extends Server {
     return this;
   }
 
+  configureEtasuEndpoints() {
+    const db = Globals.database;
```

Comment:
Is there anyway we can split-up this huge method and move it into an ETASU specific file?